python/pyproject.toml

```diff
@@ -0,0 +1,16 @@
+[project]
+name = "solana-trader-python-proto"
```

Comment:
minor: we could probably remove `python` from this name for brevity, since `pypi` projects are Python and we can be more brief. Also, since our other project is `bxsolana-trader-api`, we could also make this `bxsolana-trader-proto`?